### PR TITLE
sdk logging file size & rotation

### PR DESCRIPTION
**Pull request by kobymeir**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-8034

## Description
sdk logging file size & rotation

Content PR - https://github.com/demisto/content/pull/29144

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 ## Unreleased
 * Added a period at the end of lines produced by the **generate-docs** command that state the tested version of the product.
 * Update `RN112` validation's docs reference link.
+* Added support to control the maximum file size and log rotation files count in the sdk logger.
+* Fixed an issue with wrong recommendation for log level deprecated arguments in the sdk logger.
+* Fixed an issue with wrong documentation for log level arguments in the sdk logger.
```
Comment (dorschw) [inline CHANGELOG.md:7]: see the end [here](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs)

File: demisto_sdk/commands/common/logger.py [hunk 2]
```diff
@@ -25,18 +26,42 @@
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
 
 DEPRECATED_PARAMETERS = {
```
Comment (dorschw) [inline demisto_sdk/commands/common/logger.py:28]: were these underscored before deprecation?

**Review by dorschw — COMMENTED** [review]
Why `kebab-case` rather than `snake_case`?

File: demisto_sdk/commands/common/logger.py (hunk 2 above)
Comment (kobymeir) [reply demisto_sdk/commands/common/logger.py:28]: i'm just changing the new name, which use underscore and not hyphens

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 ## Unreleased
 * Added a period at the end of lines produced by the **generate-docs** command that state the tested version of the product.
 * Update `RN112` validation's docs reference link.
+* Fixed an issue where **<command>** would <bad thing> when <condition>
```
Comment (dorschw) [inline CHANGELOG.md:5]: 🤔 
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
...
+* Added support to control the maximum file size and log rotation files count in the sdk logger.
+* Fixed an issue with where passing the deprecated logging arguments to any command presented the wrong recommendation for the argument substitution.
```
Comment (dorschw) [inline CHANGELOG.md:7]:
```suggestion
* Fixed an issue with where passing the deprecated logging arguments to any command presented an incorrect recommendation for argument substitution.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
...
+* Fixed an issue with where **graph** commands logging argument documentation was wrong.
+* Fixed an issue with where **generate-unit-tests** logging argument documentation was wrong.
+* Fixed an issue with where **generate-modeling-rules** logging argument documentation was wrong.
```
Comment (dorschw) [inline CHANGELOG.md:10]:
```suggestion
* Fixed an issue where the documentation of logging arguments was incorrect.
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**